tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts

```diff
@@ -174,8 +174,13 @@ export class LineChartComponent implements AfterViewInit, OnChanges, OnDestroy {
       this.isMetadataUpdated = true;
     }
 
+    if (this.scaleUpdated) {
+      this.isViewBoxOverridden = false;
```

Comment:
Is it worth adding a test case to cover the scenario: "it updates the viewBox upon data changes, when a manual change is followed by a scale change"
?

Reply:
Done.